src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -27,15 +29,18 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
 
     /**
      * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
      * trimmed.
      * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
      */
-    public static Index parseIndex(String oneBasedIndex) throws ParseException {
+    public static Index parseIndex(String oneBasedIndex) throws CommandException, ParseException {
```

Comment:
We should only throw `ParseException` in this function.

`CommandException` is used for errors which occur during execurtion of a command.


`ParserException` Represents a parse error encountered by a parser.

In this case, the error occurs during the parsing stage when the users passes in `-1`.